docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -1,19 +1,179 @@
 implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '{current-version}')
 ----
 
+. Define your JDBC driver and add appropriate dependency. Let's take PostgreSQL as an example:
++
+- Open JDBC page on https://mvnrepository.com/open-source/jdbc-drivers[maven repository];
+- Find driver for your database and open details (https://mvnrepository.com/artifact/org.postgresql/postgresql[PostgreSQL JDBC Driver]);
+- Choose and open version (https://mvnrepository.com/artifact/org.postgresql/postgresql/42.4.0[42.4.0]);
+- Go to the Gradle tab and copy implementation string;
+- Paste chosen dependency into your build.gradle file:
++
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.postgresql', name: 'postgresql', version: '42.4.0')
+----
++
+NOTE: In case Maven repository does not contain driver you need, please refer to the documentation of your database to define suitable JDBC driver.
+
 == Configuration
 
+=== Database properties
+
+For database interaction, you will need to set related properties. They have `db.connection.` prefix followed by the database name (`vividus` in the example). Come up with your own database name to use within the tests. Put properties into the `environment.properties` file:
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:dbms://databaseUrl:port/databaseName?parameter1=true&parameter2=value <1>
+db.connection.vividus.username= <2>
+db.connection.vividus.password= <2>
+db.connection.vividus.driver-class-name=<3>
+----
+<1> Database address. Consists of:
+    - jdbc: - reference to JDBC;
+    - dbms: - Database Management System, e.g. `postgresql`;
+    - //databaseUrl - address to your database;
+    - :port - port number, e.g. 5432;
+    - /databaseName - name of your database;
+    - parameter - any additional parameters you need to pass;
++
+IMPORTANT: URL structure slightly changes depending on the database management system. You can find templates for different DBMS in the xref:plugins:plugin-db.adoc#_url_templates[setup paragraph].
+<2> Credentials for given database connection;
+<3> Driver class name (Optional), e.g. `org.postgresql.Driver`. You might need it to specify the driver name for the exact database.
+
+=== Setup examples
+
+Here are some examples of setting different databases connection. Follow these steps to get everything right:
+
+. Add into the `build.gradle` provided db plugin and the driver;
+. Build the project refreshing dependenices:
++
+.Eclipse
+[source,shell]
+----
+./gradlew cleanEclipse eclipse --refresh-dependencies
+----
++
+.IDEA
+[source,shell]
+----
+./gradlew cleanIdea idea --refresh-dependencies
+----
+. Refresh IDE project (or restart IDE);
+. Copy provided properties;
+. Update URL, username and password according to your database.
+
+==== MySQL
+
+.build.gradle
+[source,shell]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '0.4.14-SNAPSHOT')
```

Comment:
I'm doubtful we need to add this line to each example, from my point of view the approach is clearly described in `Installation` section